feg/protos/s8_proxy.proto

```diff
@@ -238,7 +238,7 @@ message DeleteBearerResponsePgw {
     uint32 c_pgw_teid = 4;
     // control plane TEID given by PGW during CreateSession
     uint32 linked_bearer_id = 5;
-    BearerContext bearer_context = 6;
+    repeated BearerContext bearer_context = 6;
```

Comment:
is this safe?

Reply:
yep, we haven't even finished the feature on AGW 😭  , so nobody is using it but my tests.
So lets change it here.